src/simulator.zig

```diff
@@ -536,9 +536,14 @@ pub const Simulator = struct {
         for (simulator.cluster.replicas) |replica| {
             if (simulator.core.isSet(replica.replica)) {
                 assert(simulator.cluster.replica_health[replica.replica] == .up);
-                if (replica.commit_min < replica.op) {
-                    if (missing_op == null or missing_op.? > replica.commit_min + 1) {
-                        missing_op = replica.commit_min + 1;
+                if (replica.op > replica.commit_min) {
```

Comment:
Just curious, but why swap this order?

Reply:
No reason, just wrote this again and the order came out like this. I remember someone on our team argued once that `x < y` is always readable, but I am not sure I agree with them here: `op > commit_min` reads better, as it is an exceptional case.